modules/system/dice-wfrp4e.js

```diff
@@ -820,6 +821,10 @@ export default class DiceWFRP {
       WFRP_Utility.handleTableClick(ev)
     })
 
+    html.on('mousedown', '.travel-click', ev => {
+      TravelDistanceWfrp4e.handleTravelClick(ev)
```

Comment:
This is also for clicking on custom links like tables/pay/conditions/etc. Travel doesn't seem like it fits there

Reply:
Yes, /travel command is issuing links